src/bp/ui-admin/src/Pages/Bot/Details.jsx

```diff
@@ -466,26 +475,25 @@ class Bots extends Component {
   }
 
   renderPictures() {
+    // [TODO] max.cloutier 2020.01.17 functional, but the style of this section should be revisited
     return (
       <Fragment>
         <Row>
-          <Col md={7}>
+          <Col md={6}>
             <Label>
               <strong>Bot Avatar</strong>
             </Label>
             <Input type="file" accept="image/*" name="avatarUrl" onChange={this.handleImageFileChanged} />
+            {this.state.avatarUrl && <img height={75} alt="avatar" src={this.state.avatarUrl} />}
           </Col>
-          <Col md={4}>{this.state.avatarUrl && <img height={75} alt="avatar" src={this.state.avatarUrl} />}</Col>
-        </Row>
-        <Row>
-          <Col md={4}>
+          <Col md={6}>
             <Label>
               <strong>Cover Picture</strong>
             </Label>
             <Input type="file" accept="image/*" name="coverPictureUrl" onChange={this.handleImageFileChanged} />
-          </Col>
-          <Col md={8}>
-            {this.state.coverPictureUrl && <img width="100%" alt="cover" src={this.state.coverPictureUrl} />}
+            {this.state.coverPictureUrl && (
+              <img style={{ width: 'auto', maxWidth: '100%' }} alt="cover" src={this.state.coverPictureUrl} />
```

Comment:
we try to prefer css class over to inline style but there might be a reason why you went for inline style ? If yes, I'd like to hear you out :)

Reply:
No reasons, the image had width="100%" previously so I didn't overthink it, I'll address it